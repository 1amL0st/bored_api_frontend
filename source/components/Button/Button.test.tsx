import React from 'react';
import renderer from 'react-test-renderer';

import { Button } from './index';

describe('My Test', () => {
  test('Button test', () => {
    const component = renderer.create(<Button>Button</Button>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('Button with child div', () => {
    const component = renderer.create(
      <Button>
        <div>Hello</div>
      </Button>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
