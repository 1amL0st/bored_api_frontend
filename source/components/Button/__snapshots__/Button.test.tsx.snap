// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My Test Button test 1`] = `
<button
  className="button"
>
  Button
</button>
`;

exports[`My Test Button with child div 1`] = `
<button
  className="button"
>
  <div>
    Hello
  </div>
</button>
`;
